tfalade423/centralPay
# failure and successful template

### What Does This PR Do?
- It shows when a transaction is successful or failed


#### Description of Task to be completed
- [x] Failure Notification Template
- [x]  Success Notification Template


#### How should this be manually tested?
Pull this repo and do the following:
open the HTML in the browser you will see both failure and success 

#### Any background context you want to provide?
N/a


#### What are the relevant Jira Issue Numbers?
https://nibss.atlassian.net/browse/SKP-335

#### Screenshots (if appropriate)
N/a
